### PR TITLE
Vue 3  module federation implementation 

### PR DESCRIPTION
**Pull request by anish2690**

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
Please include a readme file :)

**Comment by ScriptedAlchemy** [comment]
Also opened this PR back to your example https://github.com/anish2690/module-federation-examples/pull/1/files

**Comment by anish2690** [comment]
> Please include a readme file :)

updated

File: vue3-demo/home/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+module.exports = (env = {}) => ({
+  mode: "development",
+  cache: false,
+  devtool: "source-map",
+  optimization: {
+    minimize: false,
+  },
+  target: "web",
+  entry: path.resolve(__dirname, "./src/main.js"),
+  // output: {
```
Comment (ScriptedAlchemy) [inline vue3-demo/home/webpack.config.js:15]: lets delete any commented out code and clean this up a little now that its final

File: vue3-demo/layout/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+module.exports = (env = {}) => ({
+  mode: "development",
+  cache: false,
+  devtool: "source-map",
+  optimization: {
+    minimize: false,
+  },
+  target: "web",
+  entry: path.resolve(__dirname, "./src/main.js"),
+  // output: {
```
Comment (ScriptedAlchemy) [inline vue3-demo/layout/webpack.config.js:15]: same here

**Pull request merged**